src/components/calltree/CallTree.js

```diff
@@ -233,14 +233,21 @@ class CallTreeImpl extends PureComponent<Props> {
     );
   };
 
-  _onKeyDown = (
-    event: SyntheticKeyboardEvent<>,
-    callNodeIndex: IndexIntoCallNodeTable | null
-  ) => {
-    if (callNodeIndex !== null) {
-      const { handleCallNodeTransformShortcut, threadsKey } = this.props;
-      handleCallNodeTransformShortcut(event, threadsKey, callNodeIndex);
+  _onKeyDown = (event: SyntheticKeyboardEvent<>) => {
+    const {
+      selectedCallNodeIndex,
+      rightClickedCallNodeIndex,
+      handleCallNodeTransformShortcut,
+      threadsKey,
+    } = this.props;
+    const nodeIndex =
+      rightClickedCallNodeIndex === null
+        ? selectedCallNodeIndex
+        : rightClickedCallNodeIndex;
```

Comment:
I use this ternary everywhere, with the if condition below, rather than a possibly more simple:
```js
if (selectedCallNodeIndex !== null || rightClickedCallNodeIndex !== null) {
  handleCallNodeTransformShortcut(
    event,
    threadsKey,
    rightClickedCallNodeIndex === null
        ? selectedCallNodeIndex
        : rightClickedCallNodeIndex
  );
}
```
because Flow wasn't happy with this way.